app/core/Encryptor/pbkdf2.ts

```diff
@@ -0,0 +1,32 @@
+import { bytesToString, hexToBytes } from '@metamask/utils';
+import { NativeModules } from 'react-native';
+import { ShaAlgorithm } from './constants';
+
+/**
+ * Derives a key using PBKDF2.
+ *
+ * @param password - The password used to generate the key.
+ * @param salt - The salt used during key generation.
+ * @param iterations - The number of iterations used during key generation.
+ * @param keyLength - The length of the key to generate.
```

Comment:
Just to be extra explicit here.

```suggestion
 * @param keyLength - The length (in bytes) of the key to generate.
```